### ISSUE TITLE
Nothing to see at http://localhost:9090

### ISSUE THREAD

**Issue by pavelbinar** [mafintosh/peerwiki#8]
Hi,

I am trying to test it out...
I successfully installed the app globally via: `npm install -g peerwiki` and then run via: `DEBUG=peerwiki peerwiki 9090` or `DEBUG=peerwiki peerwiki 9090 --use-index` in either way I do not see anything in Google Chrome at: [http://localhost:9090/BitTorrent](http://localhost:9090/BitTorrent) or [http://localhost:9090](http://localhost:9090)
Getting `Error code: ERR_CONNECTION_REFUSED`.

Here is the log:

```
EBUG=peerwiki peerwiki 9090
Verifying previously downloaded data...
  peerwiki engine is ready +0ms
  peerwiki fetching required wikipedia header... +2ms
  peerwiki connected to 0/0 (0 B/s) peers +5s
  peerwiki connected to 0/0 (0 B/s) peers +5s

and so on...
```

**Comment by mafintosh** [comment]
Does it keep saying 0/0 peers?

**Comment by pavelbinar** [comment]
Yes, even if running several minutes, still getting just: `peerwiki connected to 0/0 (0 B/s) peers +5s`

**Comment by pavelbinar** [comment]
BTW: I am not using any firewall and I am able to download torrents via programs line µTorrent.

**Comment by MaxPleaner** [comment]
Same issue
